extract_msg/message.py

```diff
@@ -33,9 +37,9 @@ def __init__(self, path, prefix='', attachmentClass=Attachment, filename=None):
         :param filename: optional, the filename to be used by default when saving.
         """
         # WARNING DO NOT MANUALLY MODIFY PREFIX. Let the program set it.
-        if debug:
+        if debug._debug:
```

Comment:
@TheElementalOfCreation The if debug is redundant. logger.debug will only fire if the log level is set to debug.